SteerCo Charter Proposal (#144)
finos/common-cloud-controls: docs/formation/bootstrap.md

# Governance Bootstrap

In March 2023, the CCC community identified a need for improved organizational structure to better enable contributions to the standard.

This improvement work is being done in two phases, detailed below.

Because this work includes the creation of strict charters and limitations on future change, it will only happen this way once. All subsequent changes must follow steps outlined in the respective group charters.

# Phase 1 — Steering Committee Bootstrapping

> The first, and most important step, is to create a charter and assign members to the top level organizational unit: the Steering Committee. This must be done through a consensus of the most active CCC participants, with an opportunity for input from the entire community.

Reviewer: @eddie-knight - I have a concern with: "...a consensus of the most active CCC participants". The steerco will become the standard's oversight body and to have credibility with regulators it needs to be Financial Services. Broader community input is critical (e.g. from vendors, CSPs, etc), but we need Banks/FinServ overseeing it to ensure regulators will take it seriously.

Reviewer 2: I agree with this. 💯

Reviewer 3: Agree. We need to ensure this is open to input from the broader community. But from the standards and regulatory perspective it would need to be focussed on financial institutions to gain that credibility from regulators. The FI's also have that deep understanding of the regulatory scope to help guide the project.

Author: changing to _most active CCC participants **from financial service institutions**_

Author: changing to "_most active CCC participants **from financial service institutions**_"

Process:

1. A proposed charter will be presented to an open-invite irregular call
1. Inaugural committee members will be nominated by the members of the same call
1. The greater community will be notified via ccc-participants+subscribe@lists.finos.org

> 1. A feedback period will be held for seven (7) days, during which time anyone may give feedback

Author: conveniently we've gone over 2 weeks on this feedback period now, so this is an easy change

# Governance Bootstrap

In March 2024, the CCC community identified a need for improved organizational structure to better enable contributions to the standard.

> This improvement work is being done in two phases, detailed below.

Reviewer 4: ```suggestion
This improvement work is being done in three phases, detailed below.
```

Because this work includes the creation of strict charters and limitations on future change, it will only happen this way once. All subsequent changes must follow steps outlined in the respective group charters.

# Phase 1 — Governance Bootstrapping

The first, and most important step, is to create a charter and assign members to the top level organizational unit: the Steering Committee. This must be done through a consensus of the most active CCC participants from financial service institutions, with an opportunity for input from the entire community.

Process:

> 1. A proposed charter was be presented to an off-calendar call

Reviewer 4: "was be"? Should this say "A proposed charter was presented on an off-calendar call"

> 1. A proposed charter was be presented to an off-calendar call

Reviewer 4: ```suggestion
1. A proposed charter was presented on an off-calendar call
```

1. The greater community was notified via ccc-participants+subscribe@lists.finos.org
1. A feedback period was held for two (2) weeks, during which time feedback was incorporated from more than 10 community members
1. A consensus was achieved in an off-calendar call at the end of the feedback period
1. Additional fixes were incorporated for polish prior to merging

# Phase 2 — Inaugural Steering Committee Appointments

As the official election process inherently depends on an extant committee, the initial bootstrapping of the committee requires an appointing authority.

Process:

1. The greater community will be invited via ccc-participants+subscribe@lists.finos.org to self-nominate themselves by commenting on [PR #150](https://github.com/finos/common-cloud-controls/issues/150). 
1. Reminders to self-nominate will be made in the next two community meetings, and _ad hoc_ as needed.
1. The FINOS Technical Oversight Committee (TOC) will present a list of eligible nominees to the FINOS Board of Directors during their next upcoming meeting.

> 1. Upon validation of eligibility and community support, the TOC and Board will formalize the nominations by updating the project README to reflect the changes.

Reviewer 4: ```suggestion
1. Upon validation of eligibility and community support, the TOC and Board will formalize the nominations and the README will be subsequently updated to reflect said changes.
```

# Phase 3 — Organizational Bootstrapping

The Steering Committee (SC) will self-organize to define the community structure, document the processes surrounding it, and launch the inaugural community groups. The goal is to have this proccess quickly completed so that the subsequent groups are free to begin their work.

Following the procedure set out in the SC charter:

1. The SC will determine their exact scope of work and set a meeting cadence for the bootstrapping phase.
1. The SC will define and document the intended community structure, including all group types, member roles, and accountability structure.
1. The SC will set the charters for any inaugural groups, then appoint leadership as needed.
1. The SC will operate at a heightened awareness for the next two months, seeking to iterate on the community organizational structure and/or support community groups as needed.

> 1. At the end of the aforementioned, the SC should scale back to allow groups to self-organize according to their respective charters.

Author: ```suggestion
1. Once established, community groups should self-organize according to their respective charters.
```
